glib-core/bd.cpp

```diff
@@ -1,15 +1,23 @@
 #if SW_TRACE
 #include <execinfo.h>
 #endif
-
+#include <math.h>
 /////////////////////////////////////////////////
 // Mathmatical-Errors
+
 #if defined(__BCPLUSPLUS__) && (__BCPLUSPLUS__==0x0530)
 int std::_matherr(struct math_exception* e){
   e->retval=0;
   return 1;
 }
 #elif defined(GLib_GLIBC) || defined(GLib_BSD)
+struct __exception {

```

Comment:
as per #146

Reply:
Thanks for the fix.

Reply:
No worries. Do you have any idea how the rest of the review is going to work, do I need to poke anyone in particular or just wait for it?

Reply:
Oh I have not contributed to SNAP, so I'm not sure. But I had to do this patch myself and this shed some more light for me.